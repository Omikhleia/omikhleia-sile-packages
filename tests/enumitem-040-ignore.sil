\begin[papersize=a6, class=book]{document}
\script[src=packages/enumitem]
\font[family=Libertinus Serif]% Default Gentium lacks the white circle.
\nofolios
\neverindent
% Test case 040: text is ignored inside list structured tags (warning only)

\begin{itemize}
 This text shall be ignored
    \item{Level 1 Item}
 This text shall be ignored
    \item{Level 1 Item}
 This text shall be ignored
\end{itemize}

\end{document}
