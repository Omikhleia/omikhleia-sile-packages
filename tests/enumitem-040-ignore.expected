Set paper size 	297.6377985	419.5275636
Begin page
Mx 	30.4489
My 	55.6452
Set font 	Libertinus Serif;10;400;;normal;;LTR
T	1768 w=3.5100	(•)
Mx 	39.7039
Mx 	5.2800
Mx 	4.4000
Mx 	4.8900
Mx 	4.4700
Mx 	2.6400
T	45 a=5.2800 70 a=4.4700 87 a=4.9700 70 a=4.4700 77 a=2.6400	(Level)
Mx 	64.4046
T	18 w=4.6500	(1)
Mx 	72.0753
T	42 85 70 78 w=18.5000	(Item)
Mx 	30.4489
My 	67.6452
T	1768 w=3.5100	(•)
Mx 	39.7039
Mx 	5.2800
Mx 	4.4000
Mx 	4.8900
Mx 	4.4700
Mx 	2.6400
T	45 a=5.2800 70 a=4.4700 87 a=4.9700 70 a=4.4700 77 a=2.6400	(Level)
Mx 	64.4046
T	18 w=4.6500	(1)
Mx 	72.0753
T	42 85 70 78 w=18.5000	(Item)
End page
Finish
